apps/frontend/src/libs/components/input/styles.module.css

```diff
@@ -54,6 +54,21 @@
 	resize: none;
 }
 
+.input-checkbox {
+	width: 14px;
+	height: 14px;
+	padding: 0;
+	margin: 0 0 0 5px;
```

Comment:
Common components shouldn't have margins
```suggestion
```

Reply:
I have to remove it, we no longer have `.input-checkbox`.